### ISSUE TITLE
Allow specification of colors with alpha-transparency for `workbench.colorCustomizations`.

### ISSUE THREAD

**Issue by mikebronner** [microsoft/vscode#199968]
This is not a duplicate of #134213. I am not looking to make my entire theme transparent, etc.

I would like to be able to override these diff settings, regardless of theme, to make them easier to read. Allowing for transparency would allow the color to blend with the theme background, and not replace it completely with some arbitrary solid color that might completely conflict with the theme (light vs. dark themes, for example).

I would like to be able to do something like this:
```json
    "workbench.colorCustomizations": {
        "diffEditor.insertedTextBackground": "#00ff0080",
        "diffEditor.removedTextBackground": "#ff000080",
        "diffEditor.insertedLineBackground": "#00ff0040",
        "diffEditor.removedLineBackground": "#ff000040",
    },
```

**Comment by hediet** [comment]
I'm quite certain these colors support transparency, no?

**Comment by mikebronner** [comment]
@hediet thank you for following up. In trying to elaborate more, I discovered why I thought this was the case: the examples I was looking at, and the transparency settings overlapping each other, made it seem like all changed code was bright green.

In playing around with the colors and testing different diff examples, I found that you are correct, it is actually working as intended. Closing this now.
